akka-docs/rst/java/persistence.rst

```diff
@@ -385,6 +385,10 @@ Deleting messages in event sourcing based applications is typically either not u
 up until the sequence number of the data held by that snapshot can be issued to safely delete the previous events
 while still having access to the accumulated state during replays - by loading the snapshot.
 
+.. warning::
+  If you are using :ref:`persistence-query-java`, query results will be missing deleted messages in a journal.
```

Comment:
it depends on the journal implementation, I think the leveldb implementation is a completely detached materialized view. Reword to "may be missing deleted messages in a journal, depending on how deletions are implemented in the journal plugin"